src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private final int height;
+    private final int leg;
+
+    public RightTriangle(String color, int height, int leg) {
+        super(color);
+        this.height = height;
+        this.leg = leg;
+    }
+
+    @Override
+    public String draw() {
+        return "Figure: RightTriangle, height: " + height + ", leg: " + leg + super.draw();
```

Comment:
The `draw()` method should only be responsible for drawing the figure, not for providing its dimensions. Consider separating the responsibility of drawing from the responsibility of reporting the dimensions of the figure.